msi/release.sh

```diff
@@ -52,7 +135,7 @@ BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
             </Component>
             <Component Directory="BinFolder">
                 <File Source="mingw64\\share\\git\\compat-bash.exe"
-                      Name="sh.exe" />
+		      Name="sh.exe" />
```

Comment:
This was to make the whitespace consistent. The line was indented with two tabs and a few spaces when it should have been all spaces, as all other indenting within the here-document is indented with spaces for generating the `GitComponents.wxs` WiX XML file.